great_expectations/render/renderer_configuration.py

```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass, field
+from typing import Union
+
+from great_expectations.core import (
+    ExpectationConfiguration,
+    ExpectationValidationResult,
+)
+
+
+@dataclass(frozen=True)
+class RendererConfiguration:
+    """Configuration object built for each renderer."""
+
+    configuration: Union[ExpectationConfiguration, None]
+    result: Union[ExpectationValidationResult, None]
+    language: str = "en"
```

Comment:
`en` is based off of [ISO 639-1 language codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes). An `Enum` based on that list could be valuable. This field is not currently being used anywhere in the open source codebase, but may be in the future. Regardless of GX use, the `language` parameter exists in all renderer signatures, so removing support could break custom renderers.